flow/scripts/final_report.tcl

```diff
@@ -83,6 +80,9 @@ if {[info exist ::env(RCX_RULES)]} {
 source $::env(SCRIPTS_DIR)/report_metrics.tcl
 report_metrics "finish"
 
+write_def $::env(RESULTS_DIR)/6_final.def
+write_verilog $::env(RESULTS_DIR)/6_final.v
+
```

Comment:
Maybe this is just me, but it'd be nice to write out the files as early as possible in case e.g. OpenRCX errors out and aborts the script.

Reply:
I see your point. I was trying to make things more uniform and reduce the possibility of people adding code after the write commands that change the design. Fixed on 4dc1928